python/pyspark/mllib/clustering.py

```diff
@@ -843,7 +843,7 @@ def setInitialCenters(self, centers, weights):
     @since('1.5.0')
     def setRandomCenters(self, dim, weight, seed):
         """
-        Set the initial centres to be random samples from
+        Set the initial centers to be random samples from
```

Comment:
if this is an en-GB thing, I'll drop (I try not to enforce a particular style; this repo appears very mixed; I think this change was done by an automated pass)